myfitnesspal/client.py

```diff
@@ -220,6 +232,55 @@ def get_date(self, *args, **kwargs):
 
         return day
 
+    def get_measurements(self, measurement='Weight', start=None, end=None):
+        if start is None:
```

Comment:
I can see the argument both ways. I view check-ins as a retrospective, where your start day is your current physique, and you look back on your progress. Alternatively, I could just make them two generic dates, and anything outside of the range won't get returned. Thoughts?